src/main/java/TaskManager.java

```diff
@@ -0,0 +1,100 @@
+public class TaskManager {
+
+    public static final String TASK_ADDED = "Got it. I've added this task:\n";
+    public Task[] taskList = new Task[100];
+    public int numOfTasks = 0;
+
+    /**
+     * Add a task into the list and prints an indication it has been added successfully
+     *
+     * @param task task to add
+     */
+    public void addTask(Task task) {
+        taskList[numOfTasks] = task;
+        numOfTasks++;
+        System.out.println(
+                Duke.PRINT_LINE
+                        + TASK_ADDED
+                        + task + "\n"
+                        + "Now you have " + numOfTasks + " in the list.\n"
+                        + Duke.PRINT_LINE
+        );
+    }
+    /**
+     * Adds a todo type task to the list
+     *
+     * @param input input of user
+     */
+    public void addToDo(String input) {
+        String todoTask = input.substring("todo".length(), input.length());
```

Comment:
Avoid complicated expressions. Please consider extracting the logic into another method, which can also make the code more readable. Alternatively, you can also consider splitting the logic into multiple lines of code.